### PR TITLE
Coding style update - use // for single line comments

### PR DESCRIPTION
**Pull request by bluewavenet**
Signed-off-by: Rob White <rob@blue-wave.net>

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -191,12 +193,14 @@ init_signals(void)
...
-	/* Trap SIGPIPE */
+	// Trap SIGPIPE
+
 	/* This is done so that when libhttpd does a socket operation on
 	 * a disconnected socket (i.e.: Broken Pipes) we catch the signal
 	 * and do nothing. The alternative is to exit. SIGPIPE are harmless
 	 * if not desirable.
 	 */
+
 	debug(LOG_DEBUG, "Setting SIGPIPE  handler to SIG_IGN");
```
Comment (mwarning) [inline src/main.c]: an extra line sneaked in here :)

File: src/template.c [hunk 2]
```diff
@@ -47,20 +47,21 @@ int tmpl_parse(struct template *vars, char *dst, size_t dst_len, const char *src
 			continue;
 		}
 
-		/* we know it's a '$'. But we are interest in the next char */
```
Comment (mwarning) [inline src/template.c]: Let's fix typos in a separate commit.

File: src/main.c (hunk 1 above)
Comment (bluewavenet) [reply src/main.c]: It was deliberate ;-)
As it is an inline comment, the extra line adds to visual balance and readability in my opinion.

File: src/template.c (hunk 2 above)
Comment (bluewavenet) [reply src/template.c]: Maybe, but I would prefer to fix little typos in comments as I spot them, otherwise I am sure to forget them later.

**Pull request merged**